mfkdf2/src/derive/factors/stack.rs

```diff
@@ -1,32 +1,132 @@
-use std::{collections::HashMap, rc::Rc};
+use std::collections::HashMap;
+
+use serde::{Deserialize, Serialize};
+use serde_json::{Value, json};
 
 use crate::{
-  derive::{DeriveFactorFn, factors::MFKDF2DerivedFactor},
-  error::MFKDF2Result,
-  setup::key::Policy,
+  definitions::mfkdf_derived_key::MFKDF2DerivedKey,
+  derive::FactorDerive,
+  error::{MFKDF2Error, MFKDF2Result},
+  policy::Policy,
+  setup::factors::{Factor, FactorType, MFKDF2Factor, stack::Stack},
 };
+#[derive(Clone, Debug, Serialize, Deserialize, uniffi::Record)]
+pub struct DeriveStack {
+  pub factors: HashMap<String, MFKDF2Factor>,
+  pub key:     MFKDF2DerivedKey,
+}
```

Comment:
Why do we need a `DerivedVersion` of this?

Reply:
that's a mistake. removed in #33